dhash/module.py

```diff
@@ -1,13 +1,16 @@
 import asyncio
 import dhash
+import re
 import time
+import urllib.request
```

Comment:
We're trying to support `aiohttp` as our HTTP library of choice, because it works nice with async code. If updating this PR to use it instead would be too much work, we can do that in the future.

Reply:
I can rewrite that later if you wish, but it would request bigger changes and another bunch of testing.